### ISSUE TITLE
MarkdownStyleSheet blockquote doesn't work.

### ISSUE THREAD

**Issue by ji3g4m6zo6** [flutter/flutter_markdown#94]
Hello, I'm trying to change the font color in blockquote. 
```
MarkdownBody(
        data: data,
        styleSheet: MarkdownStyleSheet.fromTheme(Theme.of(context)).copyWith(
          blockquote: TextStyle(color: Colors.red),
        ),
      );
```

It doesn't change anything.

**Comment by SaxPlayer** [comment]
Same problem. Other style changes work fine including changing the link style in the snippet below, but the blockquote change in the same line does nothing and keeps everything including the default blue background.

`body: Center(
            child: new Markdown(data: widget.item.text,
            styleSheet: MarkdownStyleSheet.fromTheme(Theme.of(context)).copyWith(a: Theme.of(context).textTheme.body1.copyWith(fontSize: 18.0, color: Colors.black, fontWeight: FontWeight.bold), blockquote: Theme.of(context).textTheme.body1.copyWith(backgroundColor: Colors.transparent, fontSize: 22.0, color: Colors.black, fontWeight: FontWeight.bold, fontStyle: FontStyle.italic)),
            onTapLink: _onTapLink),
          ),`

**Comment by ThinkDigitalSoftware** [comment]
Wow, came to post this as well and you used the same example I did, lol. This is not fixed still in v3.0
